libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFSmartSyncPlugin/SFSmartSyncPlugin.m

```diff
@@ -73,25 +83,27 @@ - (void)resetSyncManager
 }
 
 
-- (void)handleSyncUpdate:(SFSyncState*)sync
+- (void)handleSyncUpdate:(SFSyncState*)sync isGlobal:(BOOL)isGlobal
 {
     dispatch_async(dispatch_get_main_queue(), ^{
         NSError *error = nil;
-        NSData *syncData = [NSJSONSerialization dataWithJSONObject:[sync asDict]
-                                                       options:0 // non-pretty printing
-                                                         error:&error];
+        NSMutableDictionary* detailDict = [NSMutableDictionary dictionaryWithDictionary:[sync asDict]];
+        detailDict[kSyncIsGlobalStoreArg] = isGlobal ? @YES : @NO;
```

Comment:
Sync id is not enough, we need to let JS know whether it was the global sync manager or the other sync manager that fired the event